### PR TITLE
"Edited some pages and created a needed test"

### PR DESCRIPTION
**Pull request by KatarinaStojilkovic**

### CODE REVIEW

**Comment by KatarinaStojilkovic** [comment]
Added some changes to the Sign up test.
Deleted unnecessary pages.

File: page-objects/signUpPage.ts [hunk 1]
```diff
@@ -8,4 +10,20 @@ readonly page: Page
         this.page = page
     }
 
+async createUser (){
```
Comment (testinginternship) [inline page-objects/signUpPage.ts]: Please add comments for following steps

File: tests/signUpTest.spec.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+test.beforeEach(async({page}) =>{
+    await page.goto('https://demo.prestashop.com/#/en/front')
+})
+
+test ('SignUpPage', async({page}) => {
+    
+    const pm = new PageManager(page)
+    await pm.onSignUpPage().createUser()
+
```
Comment (testinginternship) [inline tests/signUpTest.spec.ts]: What are you testing? How can we be sure that this test works?
Also, please add comments.

File: page-objects/signUpPage.ts (hunk 1 above)
Comment (testinginternship) [reply page-objects/signUpPage.ts]: Can you please add comments for this locators?

File: tests/signUpTest.spec.ts [hunk 3]
```diff
@@ -0,0 +1,98 @@
+import {test, expect} from '@playwright/test'
+import { PageManager } from '../page-objects/pageManager'
+import { faker } from '@faker-js/faker';
+import { assert } from 'console';
```
Comment (testinginternship) [inline tests/signUpTest.spec.ts]: Please remove unnecessary imports.

File: tests/signUpTest.spec.ts [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+// Assertion of the sign in button in iFrames
+
+// test ('Assertion of the Sign In', async ({page}) =>{
+//     const signInButton = page.frameLocator('#framelive').locator('#_desktop_user_info').getByText('Sign in')
+//     await signInButton.textContent()
+//     await expect(signInButton).toContainText('Sign in')
+  
+// })
+
+test ('User can register with all valid credentials', async({page}) => {
```
Comment (testinginternship) [inline tests/signUpTest.spec.ts]: Please, move all locators to the Page class.

File: tests/signUpTest.spec.ts [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+import { assert } from 'console';
+import { signUpPage } from '../page-objects/signUpPage';
+import { sign } from 'crypto';
+
+
+test.beforeEach(async({page}) =>{
+    await page.goto('https://demo.prestashop.com/#/en/front')
+})
+
+// Assertion of the sign in button in iFrames
```
Comment (testinginternship) [inline tests/signUpTest.spec.ts]: Please remove all commented code.

File: tests/signUpTest.spec.ts [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+    await checkBox1.click()
+    const isChecked2 = await checkBox1.isChecked()
+    expect(isChecked2).toBeTruthy()
+
+    // Registration
+    const saveButton = await page.frameLocator('#framelive').getByRole('button', {name: 'Save'})
+    await saveButton.click()
+    expect(saveButton).toBeDefined()
+
+    // // Inviting a function without assertion
```
Comment (testinginternship) [inline tests/signUpTest.spec.ts]: Please, remove commented code.

File: tests/signUpTest.spec.ts [hunk 7]
```diff
@@ -0,0 +1,98 @@
...
+// test ('Assertion of the Sign In', async ({page}) =>{
+//     const signInButton = page.frameLocator('#framelive').locator('#_desktop_user_info').getByText('Sign in')
+//     await signInButton.textContent()
+//     await expect(signInButton).toContainText('Sign in')
+  
+// })
+
+test ('User can register with all valid credentials', async({page}) => {
+
+    //Verify that the user can sign up with valid credentials and that the user is signed up and on main page.
```
Comment (testinginternship) [inline tests/signUpTest.spec.ts]: Please remove all 'await' that are not necessary.

**Review by testinginternship — APPROVED** [review]
PR is ready to be merged.

**Pull request merged**